### ISSUE TITLE
How do I opt out?

### ISSUE THREAD

**Issue by rpgoldman** [ReviewNB/support#52]
If I understand the terms of your privacy policy correctly, it says that my personally identifiable information will be sent to Google, and they can do whatever they want with it.

I would prefer that this not happen, and if I had a ReviewNB account, I would close it.

But I am a member of a project group that has a ReviewNB account.  Does that mean that by virtue of being a member of this group, you get to give my personal information to Google, and I have no way to opt out?  That seems very wrong: I never agreed to have a ReviewNB account at all.

**Comment by amit1rrr** [comment]
Tl;dr - We do *NOT* send any personally identifiable information to Google

I think there are some mis-conceptions. Let me clarify,

- We use Google Analytics service on our [homepage](https://www.reviewnb.com/) and NOT inside our [application](https://app.reviewnb.com/)  
- On the homepage, there is no personally identifiable information (such as your email, GitHub login or anything else).
- Google Analytics service makes use of cookies to identify unique browsing session (e.g. User 'X' has visited this site). But they do NOT have any personally identifiable information about user X. 

> But I am a member of a project group that has a ReviewNB account. Does that mean that by virtue of being a member of this group, you get to give my personal information to Google, and I have no way to opt out? 

- ReviewNB accounts are created for an individual by an individual. There is no group onboarding.
- There are a lot of GitHub orgs where only a subset of users have signed up on ReviewNB.
- So your account wouldn't have been created without your own Login/Signup action.


I hope this alleviates your concern. You can also request removal of your account from our service by emailing us (support@reviewnb.com) from your signed up email address. This deletes all the data associated with your account.

**Comment by rpgoldman** [comment]
@amit1rrr Thank you very much: yes, that does indeed resolve my worries.  I am glad it was a false alarm, but it's a symptom of having to have privacy policies that are formal, legal documents, and expecting ordinary civilians to read them.  Not that I mean to blame *you* in this regard: it's a problem with our society as a whole.

I appreciate your taking the time to clarify this, and thank you for providing ReviewNB.
